src/app/Http/Requests/ValidateEventUpdate.php

```diff
@@ -4,6 +4,15 @@
 
 class ValidateEventUpdate extends ValidateEventStore
```

Comment:
see if you can consolidate the two requests without making compromises. If not, leave it as it is

Reply:
I tried to do it, but I didn't succeed, because Update has extra rules